ql/pom.xml

```diff
@@ -1141,6 +1141,12 @@
                     <exclude>META-INF/versions/*/module-info.class</exclude>
                   </excludes>
                 </filter>
+                <filter>
```

Comment:
Apparently this fixes the compile-time problem I faced here:
https://issues.apache.org/jira/browse/HIVE-28882?focusedCommentId=17941008&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17941008

looks good for the first sight

Reply:
Yes, I'm not sure about that, was also asking myself the same question..